fastlane_bot/events/managers/pools.py

```diff
@@ -282,11 +283,7 @@ def add_pool_info(
             pool_info.update(other_args)
 
         # Update cid if necessary
-        unique_id = pool_info["descr"] if pool_info['exchange_name'] not in self.cfg.CARBON_V1_FORKS else (
-            f"{pool_info['exchange_name']} {pool_info['strategy_id']} {pool_info['fee']}"
-        )
-        pool_info["cid"] = self.pool_cid_from_descr(self.web3, unique_id)
-
+        pool_info["cid"] = get_pool_cid(pool_info, self.cfg.CARBON_V1_FORKS)
```

Comment:
Function `get_pool_cid` calls function `get_cid_base`.

And function `get_cid_base` handles the CIDs differently for carbon v1 forks and for all other cases:
```
    return (
            args["exchange_name"]
            + " "
            + str(args["strategy_id"])
    ) if args["exchange_name"] in carbon_v1_forks else (
            args["exchange_name"]
            + " "
            + args["pair_name"]
            + " "
            + str(args["fee"])
    )
```
Are you sure that this doesn't yield the exact same problem?


Module pools / Class PoolManager / Function pool_descr_from_info
Module pools / Class PoolManager / Function add_pool_info
Module utils / Global Function get_cid_base